crypto/vrf/vrf.go

```diff
@@ -16,6 +16,7 @@ type Output []byte
 type vrfEd25519 interface {
 	Prove(privateKey []byte, message []byte) (Proof, error)
 	Verify(publicKey []byte, proof Proof, message []byte) (bool, error)
+	ProofLength() int
```

Comment:
It should be named `ProofSize`, like `PubKeySize` used in other code. Size is more appropriate here than length as it represents the size of the space.

It might be better to be able to refer to the `ProofSize` directly, rather than to create a new function that only returns a constant one since it's not a variable. Which do you think it's better to do it the way now or to define `const ProofSize = xxx` in each `vrf_xxx.go` module?